x/dymns/types/market.go

```diff
@@ -0,0 +1,55 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	dymnsutils "github.com/dymensionxyz/dymension/v3/x/dymns/utils"
+	"github.com/dymensionxyz/gerr-cosmos/gerrc"
+)
+
+const (
+	// TypeName is an alias variable of AssetType_AT_DYM_NAME
+	TypeName = AssetType_AT_DYM_NAME
+
+	// TypeAlias is an alias variable of AssetType_AT_ALIAS
+	TypeAlias = AssetType_AT_ALIAS
+)
+
+var assetTypeFriendlyStrings = map[AssetType]string{
+	AssetType_AT_DYM_NAME: "Dym-Name",
+	AssetType_AT_ALIAS:    "Alias",
+}
+
+func (x AssetType) FriendlyString() string {
+	if s, ok := assetTypeFriendlyStrings[x]; ok {
+		return s
+	}
+	return "Unknown"
+}
```

Comment:
I think `Pretty()` is the typical function name here

Reply:
renamed, thanks